droptools-example/src/main/java/com/bendb/example/ExampleApp.java

```diff
@@ -30,9 +31,20 @@ public FlywayFactory getFlywayFactory(ExampleConfig configuration) {
         });
 
         bootstrap.addBundle(new JooqBundle<ExampleConfig>() {
+
             @Override
             public DataSourceFactory getDataSourceFactory(ExampleConfig configuration) {
-                return configuration.dataSourceFactory();
+                // override this method when there's only a single database configuration
+                return null;
+            }
+
+            @Override
+            public LinkedHashMap<String,DataSourceFactory> getDataSourceFactories(ExampleConfig configuration) {
```

Comment:
Rather than exposing a LinkedHashMap directly, probably want to use the SortedMap interface instead.